t/Util.pm

```diff
@@ -515,4 +516,36 @@ sub spawn_h2_server {
     return $server;
 }
 
+sub find_blackhole_ip {
+    my %ips;
+    my $port = $_[0] || 23;
+    my $blackhole_ip = undef;
+    my $poll = IO::Poll->new();
+    my $start = [ Time::HiRes::gettimeofday() ];
```

Comment:
We generally import functions from Time::HiRes. Maybe we can follow that convention by importing `gettimeofday` and doing `$start = [gettimeofday()]` (omitting the package name), or import `time` and do `$start = time` (obtain time in floating point number), using floating point maths in the code below.